cmd/runtimecfg/node-ip.go

```diff
@@ -209,7 +209,9 @@ func getSuitableIPs(retry bool, vips []net.IP, preferIPv6 bool, networkType stri
 	// Enable debug logging in utils package
 	utils.SetDebugLogLevel()
 	ipFilterFunc := utils.ValidNodeAddress
-	for {
+	const timeOutInMinutes = 5
```

Comment:
Maybe just a NIT so feel free to ignore: I see we have a whole `const` section defined at the top of this file. Could we this timeout from here to there? This way when I open the file I immediately see all the timeouts, paths and co. in a single place

Reply:
@mkowalski sounds good to me.